src/test/java/TruckTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
```

Comment:
Wrong documentation - file contains JUnit test cases for Truck.java

Reply:
Fixed